tests/test_api.py

```diff
@@ -1028,18 +1015,7 @@ def test_get_build_results(self, osbs_binary):
         ]
         flexmock(PipelineRun).should_receive('pipeline_results').and_return(pipeline_results)
 
-        assert osbs_binary.get_build_results('run_name') == {'number': 42, 'string': 'spam'}
-
-    def test_get_build_results_invalid_json(self, osbs_binary):
-        pipeline_results = [
-            {'name': 'invalid_string', 'value': 'spam'},
-        ]
-        flexmock(PipelineRun).should_receive('pipeline_results').and_return(pipeline_results)
-
-        err_msg = "invalid_string value is not valid JSON: 'spam'"
-
-        with pytest.raises(OsbsValidationException, match=err_msg):
-            osbs_binary.get_build_results('run_name')
+        assert osbs_binary.get_build_results('run_name') == pipeline_results
```

Comment:
Could you update the pipeline_results to something that could be returned by `PipelineRun.pipeline_results`?

Reply:
ok